linodecli/plugins/image-upload.py

```diff
@@ -94,6 +95,11 @@ def call(args, context):
         print("No file at {}; must be a path to a valid file.".format(filepath))
         exit(2)
 
+    # make sure it's not larger than the max upload size
```

Comment:
Could you move these conditionals up so that it'll complain about them first, before telling the user they haven't inputted a region.. Imagine a scenario where you didn't input a region, then you fix that, then you realize, the file is over 5gb. In that situation, you should have been told before anything else that the file was too big.